### ISSUE TITLE
 Chrome metric availableIncomingBitrate/googAvailableReceiveBandwidth disappeared on viewer side

### ISSUE THREAD

**Issue by skibak** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1097]
I'm updating my code from SDK release 1.2.0 to 1.5.0.
Mostly it works fine.
But I found that Chrome metric availableIncomingBitrate/googAvailableReceiveBandwidth disappeared on viewer side, and I used it for bandwidth estimation.
Can you suggest what it could be? Is it expected or some bug?
Should I use sender-side stats instead?

** Logging ** 
n/a

** Any design considerations/constraints **
In very first implementation we used transceiverOnBandwidthEstimation, but that didn't work with multiple tracks. So we switched to sending back to sender Chrome metric availableIncomingBitrate from viewer side.

**Comment by zhuker** [comment]
I found this yesterday as well. Preparing a PR to fix.
TLDR: it's a subtle bug in the library's SDP negotiation

The root cause is that we dont advertise support for REMB, but we do advertise support for TWCC.
However at this point there is no TWCC-based bandwidth estimator.

@skibak if you feel adventurous you can apply this patch to work around the issue
it disables TWCC and advertises REMB

```
diff --git src/source/PeerConnection/Rtp.c src/source/PeerConnection/Rtp.c
index ab4033206..6811ac155 100644
--- src/source/PeerConnection/Rtp.c
+++ src/source/PeerConnection/Rtp.c
@@ -294,7 +294,7 @@ STATUS writeFrame(PRtcRtpTransceiver pRtcRtpTransceiver, PFrame pFrame)
     bufferAfterEncrypt = (pKvsRtpTransceiver->sender.payloadType == pKvsRtpTransceiver->sender.rtxPayloadType);
     for (i = 0; i < pPayloadArray->payloadSubLenSize; i++) {
         pRtpPacket = pPacketList + i;
-        if (pKvsRtpTransceiver->pKvsPeerConnection->twccExtId != 0) {
+        if (FALSE && pKvsRtpTransceiver->pKvsPeerConnection->twccExtId != 0) {
             pRtpPacket->header.extension = TRUE;
             pRtpPacket->header.extensionProfile = TWCC_EXT_PROFILE;
             pRtpPacket->header.extensionLength = SIZEOF(UINT32);
diff --git src/source/PeerConnection/SessionDescription.c src/source/PeerConnection/SessionDescription.c
index ada3e7a0f..558d6eecc 100644
--- src/source/PeerConnection/SessionDescription.c
+++ src/source/PeerConnection/SessionDescription.c
@@ -621,7 +621,11 @@ STATUS populateSingleMediaSection(PKvsPeerConnection pKvsPeerConnection, PKvsRtp
     SPRINTF(pSdpMediaDescription->sdpAttributes[attributeCount].attributeValue, "%" PRId64 " nack", payloadType);
     attributeCount++;
 
-    if (pKvsPeerConnection->twccExtId != 0) {
+    STRCPY(pSdpMediaDescription->sdpAttributes[attributeCount].attributeName, "rtcp-fb");
+    SPRINTF(pSdpMediaDescription->sdpAttributes[attributeCount].attributeValue, "%" PRId64 " goog-remb", payloadType);
+    attributeCount++;
+
+    if (FALSE && pKvsPeerConnection->twccExtId != 0) {
         STRCPY(pSdpMediaDescription->sdpAttributes[attributeCount].attributeName, "rtcp-fb");
         SPRINTF(pSdpMediaDescription->sdpAttributes[attributeCount].attributeValue, "%" PRId64 " " TWCC_SDP_ATTR, payloadType);
         attributeCount++;

```

**Comment by skibak** [comment]
Thanks for prompt response!
Trying the patch now

**Comment by skibak** [comment]
Graph is back with that patch.
Thank you!

**Comment by disa6302** [comment]
@zhuker , 

Thank you for the patch. Looking forward to the PR.

**Comment by disa6302** [comment]
Closing this since the PR is merged.
